pkg/sh/net.go

```diff
@@ -28,9 +48,6 @@ func DownloadHTTP(ctx *task.Context, url string, toPath string) error {
 	if err != nil {
 		return fmt.Errorf("failed issuing GET request: %v", err)
 	}
-	if res.Body == nil {
```

Comment:
I saw this while here. I _believe_ that `res.Body` is _never_ `nil` here because `err` is guaranteed to be `nil` here. As per the document of `Do()`:
> If the returned error is nil, the Response will contain a non-nil Body which the user is expected to close.